lib/admin.php

```diff
@@ -338,5 +338,73 @@ function woocommerce_embed_page_header() {
 	</div>
 	<?php
 }
-
 add_action( 'in_admin_header', 'woocommerce_embed_page_header' );
+
+/**
+ * Registers WooCommerce specific user data to the Wordpress user API.
+ */
+function wc_admin_register_user_data() {
+	register_rest_field(
+		'user',
+		'woocommerce_meta',
+		array(
+			'get_callback'    => 'wc_admin_get_user_data_values',
+			'update_callback' => 'wc_admin_update_user_data_values',
+			'schema' => null,
+		)
+	);
+}
+ add_action( 'rest_api_init', 'wc_admin_register_user_data' );
+
+ /**
```

Comment:
There are some alignment issues with the docblocks for the next three functions. Looks like extra spaces.

If you run `composer install` and then `./vendor/bin/phpcs lib/admin.php` there will be some warnings about the spacing:

`Found precision alignment of 1 spaces.`

Reply:
Fixed in 8ba7438